CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Unused devDependencies for old docs build
 - Old `./docs` directory and out of date docs
+- Removed support for grid_code_landsat_frequency aggregation.
```

Comment:
Just double checking that this statement is accurate. From the ticket, it sounds like it's still supported, we're just removing the special handling code?

Reply:
for landsat, it's going to be supported just as the normal `grid_code_frequency` that just aggregates on `grid:code` instead of having the special aggregation `grid_code_landsat_frequency` that aggregates on a concatenation of the path and row.